src/com/pty4j/unix/PtyHelpers.java

```diff
@@ -208,14 +208,14 @@ else if (Platform.isFreeBSD()) {
     else if (Platform.isOpenBSD()) {
       myOsFacade = new com.pty4j.unix.openbsd.OSFacadeImpl();
     }
-    else if (Platform.isLinux()) {
+    else if (Platform.isLinux() || Platform.isAndroid()) {
       myOsFacade = new com.pty4j.unix.linux.OSFacadeImpl();
     }
     else if (Platform.isWindows()) {
       throw new IllegalArgumentException("WinPtyProcess should be used on Windows");
     }
     else {
-      throw new RuntimeException("Pty4J has no support for OS " + System.getProperty("os.name"));
+      throw new RuntimeException("Pty4J has no support for OS " + System.getProperty("os.name") == "Linux" ? "This is abnormal" : System.getProperty("os.name"));
```

Comment:
I suspect that `System.getProperty("os.name")` might return `Linux` in future even when `!Platform.isLinux() && !Platform.isAndroid() == true`. So, probably, it'd be better to keep the original error message (include `System.getProperty("os.name")` into the error message always).

Reply:
That's OK